logdetective/server/templates/gitlab_full_comment.md.j2

```diff
@@ -58,7 +58,10 @@ This comment was created by [Log Detective][log-detective].
 Was the provided feedback accurate and helpful? <br>Please vote with :thumbsup:
 or :thumbsdown: to help us improve.<br>
 
-
+<i>Should Log Detective report contain harmful content, please use Gitlab reporting feature
```

Comment:
I think this line is extra.

Reply:
Interesting. I've applied the patches exactly as proposed, I suppose github couldn't apply them properly.